test/e2e/cluster/add_remove_cluster.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/submariner-io/submariner/test/e2e/tcp"
 )
 
+const clusterManagementTimeoutMultiplier = 3
```

Comment:
As I was filling out the new parameter on other types of tests I figured out these disabled tests could benefit also from a multiplier.

Reply:
I'd prefer not to add another config knob, especially a hidden internal one.  I'd prefer to either set the defaults high enough to accommodate all tests or add separate user configurable knobs for various test suites as needed, eg `dataplane-connection-timeout` and `redundancy-connection-timeout`.

Reply:
ok, I'll change it to be that way, as you said, probably that's cleaner.